src/Particle/LongRange/EwaldHandlerQuasi2D.h

```diff
@@ -2,133 +2,74 @@
 // This file is distributed under the University of Illinois/NCSA Open Source License.
 // See LICENSE file in top directory for details.
 //
-// Copyright (c) 2020 QMCPACK developers.
+// Copyright (c) 2022 QMCPACK developers.
 //
-// File developed by: Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
-//                    Jeremy McMinnis, jmcminis@gmail.com, University of Illinois at Urbana-Champaign
-//                    Mark A. Berrill, berrillma@ornl.gov, Oak Ridge National Laboratory
+// File developed by: Yubo "Paul" Yang, yubo.paul.yang@gmail.com, CCQ @ Flatiron
```

Comment:
Did you create every line from scratch. If not, it is better to keep the old author list.

Reply:
I did rewrite the whole thing.
Although, I didn't intend to erase the historic contributions.
They have been restored.

Reply:
Thanks. When I read the code, "EwaldHandlerQuasi2D::evaluate_slab" before and after look very similar. So I felt better to keep the historic contributions.